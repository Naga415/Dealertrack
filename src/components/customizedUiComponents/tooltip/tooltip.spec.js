import React from "react";
import renderer from "react-test-renderer";
import Tooltip from "./tooltip";

describe("Tooltip", () => {
  it("renders <Tooltip />", () => {
    const props = {
      htmlId: "Tooltip",
      tooltipText:"Tooltip Text",
      actions: {}
    };

    const component = renderer.create(<Tooltip {...props} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
