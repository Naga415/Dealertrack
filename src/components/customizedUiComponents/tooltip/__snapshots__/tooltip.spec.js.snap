// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip renders <Tooltip /> 1`] = `
<div
  className="taxestooltip"
>
  <i
    className="fa fa-info-circle"
  />
  <span
    className="tooltiptext"
  >
    Tooltip Text
  </span>
</div>
`;
